theano/scalar/basic.py

```diff
@@ -604,12 +604,11 @@ def __sub__(self, other):
     def __mul__(self, other):
         return mul(self, other)
 
-    if PY3:
-        def __truediv__(self, other):
-            return div_proxy(self, other)
-    else:
-        def __div__(self, other):
-            return div_proxy(self, other)
```

Comment:
@lamblin @nouiz , I had tried this from outside theano without importing ```__future__``` , 

`>>> c = theano.function([a,b], b/a)
>>> c(2,5)
array(2, dtype=int32)
>>> c = theano.function([a,b], b//a)
>>> c(3,5)
array(1, dtype=int32)
>>> c(3,6)
array(2, dtype=int32)
>>> c = theano.function([a,b], b/a)
>>> c(3,6)
array(2, dtype=int32)
`